Dockerfile

```diff
@@ -1,67 +1,32 @@
-FROM frolvlad/alpine-glibc:alpine-3.10
+FROM mambaorg/micromamba:git-0f27156
 
-# much of image code ripped from
-# https://github.com/Docker-Hub-frolvlad/docker-alpine-miniconda3
 COPY BASE_IMAGE_LICENSE /
 
 LABEL maintainer="conda-forge core (@conda-forge/core)"
 
 ENV LANG en_US.UTF-8
-
-ARG CONDA_DIR="/opt/conda"
-
-ENV PATH="$CONDA_DIR/bin:$PATH"
 ENV PYTHONDONTWRITEBYTECODE=1
+WORKDIR /
+USER root
 
-# make sure the install below is not cached by docker
-ADD http://worldclockapi.com/api/json/utc/now /opt/docker/etc/timestamp
```

Comment:
Ok, I got annoyed because while it was working yesterday, today that URL is failing for me with a 503. If I add it back in, it will likely fail in the short-term.

> separated into other PRs

No problem, the logical changes are separated into individual commits, so I can easily cherry-pick them into individual PRs. Roughly how many would you like? And with any particular logical division?

> How big is the resulting image from the changes here? 

Using wagoodman/dive, my image measures 438 MB, while my build from the original is 400 MB, but I haven't tried to minimize it yet. In theory I might be able to shave off almost 90 MB if I use a separate stage to create `/opt/conda` and copy that into an alpine image, but I'm not sure if that will work.

Reply:
One pr per feature is great. The style stuff can go into a single pr for sure. 

That image size looks fine.

Reply:
Ok, I'll see what I can come up with.

In my last commit, I just reduced the image size to 356 MB :grin: